.vib/postgresql/goss/vars.yaml

```diff
@@ -0,0 +1,12 @@
+binaries:
+  - gosu
+  - postgres
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/postgresql/conf
+      - /docker-entrypoint-initdb.d
+      - /opt/bitnami/postgresql/conf/conf.d
+      - /opt/bitnami/postgresql/logs
+      - /opt/bitnami/postgresql/tmp
```

Comment:
You are missing the checks for the following directories:

- `$POSTGRESQL_VOLUME_DIR`
- `${POSTGRESQL_MOUNTED_CONF_DIR}/conf.d`
- `$POSTGRESQL_CONF_DIR`

Remember that you are not only checking the directory exists but its permissions also.